src/Microsoft.OData.Core/JsonLight/ODataJsonLightDeltaWriter.cs

```diff
@@ -290,7 +290,7 @@ void IODataOutputInStreamErrorListener.OnInStreamError()
         /// <inheritdoc/>
         Task IODataOutputInStreamErrorListener.OnInStreamErrorAsync()
         {
-            throw new NotImplementedException();
+            return this.inStreamErrorListener.OnInStreamErrorAsync();

```

Comment:
could we have a short test to test that this does get called.

Reply:
@marabooy I'll add the tests on a PR I'm working on for `ODataJsonLightOutputContext` since the logic for writing an error response is triggered from there.